plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Evaluation/Machine/Cek/Internal.hs

```diff
@@ -671,6 +669,8 @@ data Context uni fun ann
     -- ^ @[_ N]@
     | FrameAwaitFunValue !(CekValue uni fun ann) !(Context uni fun ann)
     -- ^ @[_ V]@
+    | FrameAwaitFunValues !(ArgStack uni fun ann) !(Context uni fun ann)
+    -- ^ @[[[_ V_1] ...] V_n]@
```

Comment:
Could you please rename the frame to `FrameAwaitFunValueN` or something? The current name suggests that we're expecting multiple function values, but we're expecting a single function value applied to multiple argument values.

Also, could you please use iterated application `[_ V_1 ... V_n]`, that would be more readable. And below too.

I'd also prefer us to rename `V_1`/`V_n` to `V1`/`Vn`, but given that it's existing notation, we can do that separately if we agree that it'd read better given that a standalone `_` has its own different meaning.